src/coreimage.cs

```diff
@@ -4816,67 +4827,77 @@ interface CIColorCurves {
 	[TV (11,0)]
 	[BaseType (typeof (CIFilter))]
 	interface CIDepthBlurEffect {
-		// TODO: Needs review
-		//[CoreImageProperty ("inputAperture")]
-		//// TODO: this was an NSNumber transformed to float, but maybe an int or bool is more appropriate
-		//float Aperture { get; set; }
-		//[CoreImageProperty ("inputCalibrationData")]
-		//AVCameraCalibrationData CalibrationData { get; set; }
-		//[CoreImageProperty ("inputTuningParameters")]
-		//NSDictionary TuningParameters { get; set; }
-		//[CoreImageProperty ("inputNosePositions")]
-		//CIVector NosePositions { get; set; }
-		//[CoreImageProperty ("inputLumaNoiseScale")]
-		//// TODO: this was an NSNumber transformed to float, but maybe an int or bool is more appropriate
-		//float LumaNoiseScale { get; set; }
-		//[CoreImageProperty ("inputChinPositions")]
-		//CIVector ChinPositions { get; set; }
-		//[CoreImageProperty ("inputDisparityImage")]
-		//CIImage DisparityImage { get; set; }
-		//[CoreImageProperty ("inputScaleFactor")]
-		//// TODO: this was an NSNumber transformed to float, but maybe an int or bool is more appropriate
-		//float ScaleFactor { get; set; }
-		//[CoreImageProperty ("inputRightEyePositions")]
-		//CIVector RightEyePositions { get; set; }
-		//[CoreImageProperty ("inputLeftEyePositions")]
-		//CIVector LeftEyePositions { get; set; }
-		//[CoreImageProperty ("inputFocusRect")]
-		//CIVector FocusRect { get; set; }
+		[CoreImageFilterProperty ("inputAperture")]
+		float Aperture { get; set; }
+
+		[CoreImageFilterProperty ("inputCalibrationData")]
+		AVCameraCalibrationData CalibrationData { get; set; }
+
+		// FIXME: How to bind this? Strongly typed?
+		// [CoreImageFilterProperty ("inputTuningParameters")]
+		// NSDictionary TuningParameters { get; set; }
```

Comment:
If we have documentation then let's do a strong dictionary.
If not then lets bind it as `WeakTuningParameters` so it's available/usable to developers

Reply:
So the only documentation I know being https://developer.apple.com/library/content/documentation/GraphicsImaging/Reference/CoreImageFilterReference/index.html I guess the answer is no...

Reply:
then a weak version should be exposed (it remain useful even if a strong dictionary is made available later)